utils/rust-setup.js

```diff
@@ -0,0 +1,109 @@
+const chalk = require('chalk');
+const reader = require("readline-sync");
```

Comment:
nit: wonder whether readline-sync is worth adding as dependency vs using stock `readline` from standard library

I feel like what would help is just creating `async` wrapper a-la:
https://github.com/near/near-cli/blob/4a7452f631f71bb020e25ec828d686689d1af45e/utils/eventtracking.js#L78